app/com/arpnetworking/metrics/portal/scheduling/JobCoordinator.java

```diff
@@ -237,24 +249,18 @@ private static String simpleTypeName(final Class<?> clazz) {
     @Override
     public Receive createReceive() {
         return receiveBuilder()
-                .match(AntiEntropyTick.class, message -> {
+                .matchEquals(ANTI_ENTROPY_TICK, message -> {
 
                     LOGGER.debug()
                             .setMessage("ticking")
                             .addData("repositoryType", _repositoryType)
                             .log();
-                    if (_currentlyExecuting) {
```

Comment:
This was never set to `true` and so it already had no effect.

I removed it because I am worried that we will still have the same problem if we ignore ticks. e.g. the first node reloads, causes a tick, but the job then runs anyway on a node that's still empty. Seems unlikely, but it's completely averted if we just run this for each tick that occurs.

If the actual load is an issue from running N anti-entropy ticks at startup, I could alternatively add a "batching period" to this where we delay the actual job from running after receiving the first tick.